packages/cozy-mespapiers-lib/src/components/Home/Help.jsx

```diff
@@ -0,0 +1,63 @@
+/* global cozy */
+import React, { useState } from 'react'
+import ReactMarkdown from 'react-markdown'
+
+import Button from 'cozy-ui/transpiled/react/Buttons'
+import { ConfirmDialog } from 'cozy-ui/transpiled/react/CozyDialogs'
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import IconButton from 'cozy-ui/transpiled/react/IconButton'
+
+const url = 'https://support.grandlyon.com/mes-papiers/'
+
+const Help = () => {
+  const [showDialog, setShowDialog] = useState(false)
+  const { t } = useI18n()
+  const { BarRight } = cozy.bar
+
+  const handleClick = () => {
+    window.open(url, '_blank')
+  }
+
+  return (
+    <>
+      <BarRight>
+        <IconButton onClick={() => setShowDialog(true)}>
+          <Icon icon="help" />
+        </IconButton>
+      </BarRight>
+      {showDialog && (
+        <ConfirmDialog
+          open
+          title={t('Home.Help.title')}
+          content={
+            <ReactMarkdown
+              linkTarget={href =>
+                href.includes('tel://') ? '_self' : '_blank'
+              }
+            >
+              {t('Home.Help.content', { url })}
+            </ReactMarkdown>
+          }
+          actions={
+            <>
+              <Button
+                variant="secondary"
+                label={t('Home.Help.actions.later')}
+                onClick={() => setShowDialog(false)}
+              />
+              <Button
+                label={t('Home.Help.actions.go')}
+                endIcon={<Icon icon="link-out" />}
+                onClick={handleClick}
```

Comment:
Why not using `href`? Like that right click, cmd click and so on will work.

Reply:
Parce que j'oublie tout le temps que Button peut prendre "component" en arg et donc faire un `a href` 🤦 Par contre ça casse le style, il faut que je creuse un peu plus...

Reply:
En tout cas, si on peut éviter au maximum les events JS pour la navigation, c'est mieux

Reply:
Ok oui. C'est résolu sans problème.